projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/model/enums/AtomicOrdering.java

```diff
@@ -30,18 +30,25 @@
 package com.oracle.truffle.llvm.parser.model.enums;
 
 public enum AtomicOrdering {
-    NOT_ATOMIC(0L),
-    UNORDERED(1L),
-    MONOTONIC(2L),
-    ACQUIRE(3L),
-    RELEASE(4L),
-    ACQUIRE_RELEASE(5L),
-    SEQUENTIALLY_CONSISTENT(6L);
+    NOT_ATOMIC(0L, ""),
+    UNORDERED(1L, "unordered"),
```

Comment:
please document why these enum values need a name different from the enum value name.
Recent refactorings removed these names because it was not obvious where the names are needed.